3p/frame.max.html

```diff
@@ -2,7 +2,7 @@
 <head>
 <meta charset="utf-8">
 <meta name="robots" content="noindex">
-<script src="./integration.js"></script>
+<script src="./f.js"></script>
```

Comment:
I guess we run into cross domain issues with that.

Onto another proposal;

When building the `src` of the 3pframe via `getDefaultBootstrapBaseUrl()` in `src/3p-frame.js`, we could tack on serialized configuration via the hash path.

```javascript
return 'https://' + getSubDomain(parentWindow) +
      `.${urls.thirdPartyFrameHost}/$internalRuntimeVersion$/frame.html#` +
      queryParams(urls);
```

Then, `src/config.js` can check the hash path for query parameters to deserialize and mixin to window.AMP_CONFIG